syz-cluster/pkg/blob/storage_test.go

```diff
@@ -22,10 +21,7 @@ func TestLocalStorage(t *testing.T) {
 		uris = append(uris, uri)
 	}
 	for i, uri := range uris {
-		reader, err := storage.Read(uri)
-		assert.NoError(t, err)
-		readBytes, err := io.ReadAll(reader)
-		reader.Close()
+		readBytes, err := ReadBytes(storage, uri)
```

Comment:
storage.ReadAllBytes() ?
Up to you. Generally it will work better in IDE with autocompletion.

Reply:
I've renamed the method to `ReadAllBytes()`, but I'd like it to stay a separate function (at least for now). The `Storage` interface already has a `Read()` method returns a `ReadCloser` object, which is also used in the code (e.g. to efficiently return these objects in HTTP handlers).
Forcing all `Storage` implementations to implement both `Read` and `ReadAllBytes` doesn't seem to be worth it so far.